src/resources/views/fields/video.blade.php

```diff
@@ -172,7 +172,7 @@
 
                 var fetchVimeo = function( videoId, callback ){
 
-                    var api = 'http://vimeo.com/api/v2/video/' + videoId + '.json?callback=?';
+                    var api = 'https://vimeo.com/api/v2/video/' + videoId + '.json?callback=?';
```

Comment:
I don't think so. I think HTTP doesn't complain that it's being served HTTPS links. After all, they're better :-)